doc/5_saving.md

```diff
@@ -471,6 +473,53 @@ val table = TableDef("test","words",Seq(p1Col),Seq(c1Col, c2Col),Seq(rCol))
 val rddOut = rdd.map(s => outData(s._1, s._2(0), s._2(1), s._3))
 rddOut.saveAsCassandraTableEx(table, SomeColumns("col1", "col2", "col3", "col4"))
 ```
+## Deleting rows and columns
+`RDD.deleteFromCassandra(keyspaceName, tableName)` delete rows from specified Cassandra table, using data from the RDD as primary keys.
+
+Keys RDD could be loaded from the same table:
+
+```
+sc.cassandraTable("test", "words")
+  .filter(row => ...).
+  .deleteFromCassandra("test", "words")
+```
+Or from any external source:
+
+```
+CREATE TABLE test.word_groups (group text, word text, count int,
+  PRIMARY KEY (group,word));
+```
+
+```
+sc.parallelize(Seq(("animal", "trex"), ("animal", "mammoth")))
+  .deleteFromCassandra("test", "test")
+```
+
+`deleteColumns: ColumnSelector` optional parameter allows selected columns deletion only
```

Comment:
This probably should go at the top of the section since it is describing options